hypervisor/src/arch/x86/emulator/instructions/cmp.rs

```diff
@@ -241,22 +241,14 @@ mod tests {
     use super::*;
     use crate::arch::x86::emulator::mock_vmm::*;
 
-    macro_rules! hashmap {
-        ($( $key: expr => $val: expr ),*) => {{
-            let mut map = ::std::collections::HashMap::new();
-            $( map.insert($key, $val); )*
-                map
-        }}
-    }
-
```

Comment:
For the record, `hasmap!` could be exported as a macro (`macro_export`) but it then is moved to the root of the crate. Having to write `use crate::hasmap` is not the prettiest and it's a little awkward to access it from the crate's root. So I'm ok with your change here.